time_tracker_api/time_entries/time_entries_model.py

```diff
@@ -47,3 +48,15 @@ def __init__(self):
             SQLCRUDDao.__init__(self, TimeEntrySQLModel)
 
     return TimeEntriesSQLDao()
+
+
+container_definition = {
+    'id': 'time_entry',
+    'partition_key': PartitionKey(path='/tenant_id'),
+    'unique_key_policy': {
+        'uniqueKeys': [
```

Comment:
Following the same observation of models above. I think here we need project_id, activity_id.

Reply:
No, you can be working in that same project_id and doing that same activity id even with different technologies or even the same combination of the 3 of them. For instance, you are working at something and then you change to the standup and when you finish you go back to a time entry with the same data as before.